ShoppingCart-topic1/src/MailList.java

```diff
@@ -0,0 +1,35 @@
+import java.util.ArrayList;
+
+public class MailList implements List {

```

Comment:
Maybe I should have implemented a class email service that is responsible for notifying people, and mailList invoke this operation